sdk/src/utils.ts

```diff
@@ -1,18 +0,0 @@
-export function serializeParams(params: Record<string, any>) {
```

Comment:
I believe both of these can be replaced by relying on the `URL` object. For future reference, lets try to stick to one function per file (easier to add tests later) 🙂